app/extensions/alchemy/pg_search/ingredient_extension.rb

```diff
@@ -1,18 +1,15 @@
 module Alchemy::PgSearch::IngredientExtension
+  def searchable_content
+    content_field = case type
+      when "Alchemy::Ingredients::Richtext"
```

Comment:
I think it makes sense to make this a Map as a replacement for `SEARCHABLE_INGREDIENTS`

```rb
Alchemy.searchable_ingredients = {
  "Alchemy::Ingredients::Richtext" => :stripped_body,
  "Alchemy::Ingredients::Picture" => :caption
}
```